src/app/routes/HomePage.vue

```diff
@@ -27,18 +30,68 @@ export default {
       return this.$route.query.title;
     }
   },
-  components: {
-    SearchBar,
-    Loader,
-    SearchResultItem
+
+  methods: {
+    ...mapActions(["fetchComics", "fetchRandomCharacters"]),
+
+    lazyLoad() {
+      let lazyImages = [].slice.call(document.querySelectorAll("img.lazy"));
+      let active = false;
+      if (active === false) {
+        active = true;
+        setTimeout(() => {
+          lazyImages.forEach(function(lazyImage) {
+            if (
+              lazyImage.getBoundingClientRect().top <= window.innerHeight &&
+              lazyImage.getBoundingClientRect().bottom >= 0 &&
+              getComputedStyle(lazyImage).display !== "none"
+            ) {
+              lazyImage.src = lazyImage.dataset.src;
+              lazyImage.classList.remove("lazy");
+              lazyImages = lazyImages.filter(function(image) {
```

Comment:
`lazyImages = lazyImages.filter(image => image !== lazyImage)`
I'm a bit fan of oneliners )